config/routes.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  mount GraphiQL::Rails::Engine, at: "/graphiql", graphql_path: "/graphql"
```

Comment:
Let's try to keep it consistent with Decidim at the URL levels: https://www.decidim.barcelona/api/docs

> Typically (although some particular installations may change that) you will find 3 relevant folders:
> 
>     URL/api The route where to make requests. Request are usually in the POST format.
>     URL/api/docs This documentation, every Decidim site should provide one.
>     URL/api/graphiql GraphiQL is a in-browser IDE for exploring GraphQL APIs. Some Decidim installations may choose to remove access to this tool. In that case you can use a standalone version and use any URL/api as the endpoint
>

Reply:
@andreslucena Hi!! I've added the documentation. You can see it in http://staging-bb-decidim.codegram.com/api/docs

Reply:
@andreslucena What you think about the changes? 😃

Reply:
Awesome :clap: :clap:

Reply:
Thank you!